kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -446,6 +448,15 @@
 
   @import '~kolibri.styles.definitions';
 
+  .main {
+    height: 100%;
+    // Don't put focus outline on the main div even if still in
+    // keyboard modality
+    &:focus {
+      outline-style: none !important;
+    }
```

Comment:
would you mind showing gifs of what the behavior is with and without this?

Reply:
Here's what it looks like without this override. It's because the kolibriOutline rule with keyboard modality applies to everything (has a `*` selector)

![skip link](https://user-images.githubusercontent.com/10248067/62392024-2f128c80-b51b-11e9-8305-76355a134ba5.gif)